sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -742,7 +742,7 @@ abstract class SessionCatalogSuite extends AnalysisTest with Eventually {
   private def getViewPlan(metadata: CatalogTable): LogicalPlan = {
     import org.apache.spark.sql.catalyst.dsl.expressions._
     val projectList = metadata.schema.map { field =>
-      UpCast(
+      Cast(
```

Comment:
Seems like this breaks non-ANSI build. Made a followup: https://github.com/apache/spark/pull/46614